stdlib/typing_extensions.pyi

```diff
@@ -5,6 +5,7 @@ import typing
 from _collections_abc import dict_items, dict_keys, dict_values
 from _typeshed import IdentityFunction, Incomplete
 from collections.abc import Iterable
+from types import GenericAlias, UnionType
```

Comment:
You need to put these imports behind version guards as well (and move them after the `from typing` imports)